### PR TITLE
migrate locale command

### PR DESCRIPTION
**Pull request by RainbowDashLabs**

### CODE REVIEW

File: src/main/java/de/chojo/repbot/commands/Locale.java [hunk 1]
```diff
@@ -1,73 +0,0 @@
...
-        var language = context.localizer().localizer().getLanguage(event.getOption("language").getAsString());
-        if (language.isEmpty()) {
-            event.reply(context.localize("command.locale.error.invalidLocale")).setEphemeral(true).queue();
-            return;
-        }
-        if (guilds.guild(event.getGuild()).settings().general().language(language.get())) {
-            event.reply(context.localize("command.locale.sub.set.set",
-                    Replacement.create("LOCALE", language.get().getLanguage(), Format.CODE))).queue();
-            context.commandHub().refreshGuildCommands(event.getGuild());
-            executorService.schedule(() -> context.commandHub().refreshGuildCommands(event.getGuild()), 5, TimeUnit.SECONDS);
```
Comment (Taucher2003) [inline src/main/java/de/chojo/repbot/commands/Locale.java:54]: I guess this is no longer necessary because we now let Discord handle the localization?

**Review by Taucher2003 — APPROVED** [review]

File: src/main/java/de/chojo/repbot/commands/Locale.java (hunk 1 above)
Comment (RainbowDashLabs) [reply src/main/java/de/chojo/repbot/commands/Locale.java:54]: It is still required. Guild language is only available for guild with enabled community features. Other guilds will have en-US as default. The locale is to override the guild language. If no locale is set we default to the guilds locale.

Comment (Taucher2003) [reply src/main/java/de/chojo/repbot/commands/Locale.java:54]: I meant the part with refreshing the commands, as that part is not present in the new classes. The command itself is of course still needed.

Comment (RainbowDashLabs) [reply src/main/java/de/chojo/repbot/commands/Locale.java:54]: Oh yeah. We will switch to global commands after the migration. So no more refreshing the commands

**Pull request merged**